### PR TITLE
Nisha | Test | scroll subscription table dragging

### PR DESCRIPTION
**Pull request by nisha-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 1]
```diff
@@ -696,6 +704,40 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
...
         }
     }
+    /**
+     * This will use for dragging subscription table
+     *
+     * @param {MouseEvent} e
+     * @return {*} 
+     * @memberof BuyPlanComponent
+     */
+    public move(e: MouseEvent) {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]:
```suggestion
    public move(event: MouseEvent) {
```

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 2]
```diff
@@ -696,6 +704,40 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
...
+        e.preventDefault();
+        if (!this.mouseDown) { return; }
+        const x = e.pageX - this.slider.nativeElement.offsetLeft;
+        const scroll = x -this.startX;
+        this.slider.nativeElement.scrollLeft = this.scrollLeft - scroll;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts:719]: format code

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 3]
```diff
@@ -696,6 +704,40 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
...
+    }
+    /**
+     * This will use for dragging subscription table
+     *
+     * @param {MouseEvent} e
+     * @memberof BuyPlanComponent
+     */
+    public startDragging(e: MouseEvent) {
+        this.mouseDown = true;
+        this.startX = e.pageX - this.slider.nativeElement.offsetLeft;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]:
```suggestion
        this.startX = e.pageX - this.slider.nativeElement?.offsetLeft;
```

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 4]
```diff
@@ -696,6 +704,40 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
...
+        this.scrollLeft = this.slider.nativeElement.scrollLeft;
+    }
+    /**
+     * This will use for dragging subscription table
+     *
+     * @param {MouseEvent} e
+     * @memberof BuyPlanComponent
+     */
+    public stopDragging(e: MouseEvent) {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: add return type

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 5]
```diff
@@ -704,6 +746,13 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
     */
     public ngAfterViewInit(): void {
         this.stepperIcon._getIndicatorType = () => 'number';
+
+        const sliderElement = this.slider.nativeElement;
+        console.log('Slider Element', sliderElement);
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: remove console

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts (hunk 3 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: done

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts (hunk 4 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: done

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts (hunk 5 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: done

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts:719]: done

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**